src/validation/validation_functions.jl

```diff
@@ -164,19 +164,32 @@ function validation_error(valid_info::ValidationInfo, field_value)
     return false
 end
 
+
+function validate_struct(sys::System, ps_struct::T) where T <: PowerSystemType
```

Comment:
Since you aren't using T anywhere, you should just write

`function validate_struct(sys::System, ps_struct::PowerSystemType)`